src/components/inputs/YearsCounter/YearsCounter.tsx

```diff
@@ -0,0 +1,56 @@
+import { useEffect, useState } from 'react';
+
+import './styles.css';
+
+function YearsCounter({ setCount, count }: { setCount: any; count: number }) {
+  const [years, setYears] = useState('year');
+  const [registration, setRegistration] = useState('');
+
+  useEffect(() => {
+    changeYear();
+  }, [count]);
+
+  function subtractYearsCount() {
+    if (count > 1) {
+      setCount(count - 1);
+    } else {
+      return;
+    }
+  }
+  function addYearsCount() {
+    if (count === 200) {
+      return;
+    }
+    setCount(count + 1);
+  }
+  function changeYear() {
+    const date = new Date();
+    if (count > 1) {
+      setYears(`years`);
+    } else {
+      setYears(`year`);
+    }
+    const newYear = date.getFullYear() + count;
+    setRegistration(
+      `${date.toLocaleString('default', {
+        month: 'long',
+      })} ${date.getDate()}, ${newYear}`,
+    );
+  }
+  return (
+    <div className="yearsCounterContainer">
```

Comment:
just FYI - we have a lot of component specific CSS happening. we should continue to try and create global classes and re-use them. not going to block this PR on this but something to think about when adding new components